src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
```

Comment:
Similar to the previous comment, '.body' does not refer to the body element. If you want to style the body element, you should use 'body' without the dot.